storm-server/src/main/java/org/apache/storm/DaemonConfig.java

```diff
@@ -103,6 +110,38 @@
     public static final String STORM_SCHEDULER = "storm.scheduler";
 
     /**
+     * The number of seconds that the blacklist scheduler will concern of bad slots or supervisors
+     */
+    @isInteger
+    public static final String BLACKLIST_SCHEDULER_TOLERANCE_TIME = "blacklist.scheduler.tolerance.time.secs";
+
+    /**
+     * The number of hit count that will trigger blacklist in tolerance time
+     */
+    @isInteger
```

Comment:
Is a negative count allowed?  If not we should also mark it as `@isPositiveNumber`

Same for tolerance time and resume time.

Reply:
Negative count doesn't make sense. I'll change it to `@isPositiveNumber`